pravega/src/test/scala/docs/scaladsl/PravegaGraphSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.scaladsl
+
+import akka.stream.alpakka.pravega.{ReaderSettingsBuilder, WriterSettingsBuilder}
+import akka.stream.alpakka.pravega.scaladsl.Pravega
+import akka.stream.scaladsl.{Keep, Sink, Source}
+import io.pravega.client.stream.impl.JavaSerializer
+import akka.stream.KillSwitches
+
+import scala.concurrent.{Await, Promise}
+import scala.concurrent.duration.DurationInt
+
+class PravegaGraphSpec extends PravegaBaseSpec {
+
+  val serializer = new JavaSerializer[String]
```

Comment:
I think it's better if we avoided any use of Java serialization (even in such a simple test). How about using Pravega's `UTF8StringSerializer` instead?